Doc/library/sys.rst

```diff
@@ -16,11 +16,13 @@ always available.
    On POSIX systems where Python was built with the standard ``configure``
    script, this contains the ABI flags as specified by :pep:`3149`.
 
+   .. versionadded:: 3.2
+
    .. versionchanged:: 3.8
       Default flags became an empty string (``m`` flag for pymalloc has been
       removed).
 
-   .. versionadded:: 3.2
+   .. availability:: Unix.
```

Comment:
N.B. The addition of this availability note wasn't part of the original PR; it looks like it existed on the `main` branch but not the `3.12` branch (and this was probably (at least part of) the cause of the merge conflict). However, I'm fine with adding it to the 3.12 branch here; it's true on Python 3.12 as well as Python 3.13.

Reply:
D'oh! I noticed this when doing the 3.11 backport. I came back here to fix it, only to find that it had already been merged!

Reply:
No worries!